alpha/action/merge.go

```diff
@@ -0,0 +1,264 @@
+package action
```

Comment:
Do you think it makes sense to try to keep a 1-to-1 mapping between CLI subcommands and actions? Since there isn't an `opm merge` CLI subcommand, I imagined this would be added in the `declcfg` package rather than as a separate action.